sdk/appconfiguration/azure-spring-cloud-appconfiguration-config/src/main/java/com/azure/spring/cloud/config/AppConfigurationRefresh.java

```diff
@@ -45,20 +46,28 @@
 
     private ApplicationEventPublisher publisher;
 
+    private final AppConfigurationProviderProperties appProperties;
+
     private final ClientStore clientStore;
 
     private Map<String, AppConfigurationStoreHealth> clientHealth;
 
     private String eventDataInfo;
 
+    private final Duration refreshInterval;
+
     /**
      * Component used for checking for and triggering configuration refreshes.
      * 
      * @param properties Client properties to check against.
+     * @param appProperties Library properties for configuring backoff
```

Comment:
What's the difference between properties and appProperties?

Reply:
Properties are the ones in the client config file. Our library has it's own config file for things like API version. That is appProperties.